python/lsst/sdm/schemas/lsstcam.yaml

```diff
@@ -4659,6 +4746,18 @@ tables:
     description: Error on ellipse rho (correlation coefficient) from the multiband Sersic
       model fit.
     datatype: float
+  - name: sersic_reff_major
+    datatype: float
+    description: Effective radius (major axis) from the multiband Sersic model fit, prior to convolution with the adjusted (see config) PSF value.
```

Comment:
Most users aren't going to be looking in configs, so if there's a super brief way to describe what is adjusted about the PSF that would be more helpful.